### DISCUSSION TITLE
Wärmepumpe im SunnyPortal

### DISCUSSION THREAD

**Discussion by DE89UL** [camueller/SmartApplianceEnabler#499] [Installation und Betrieb]
Moin,

erstmal danke für die tolle Umsetzung. Dank deiner ausfürlichen Erklärungen, habe ich es geschafft, die WP im Sunnyportal zu visualisieren.

Jetzt zu meinen Fragen:
-Wieso springt der Verbrauch der WP im Sunnyportal?Die WP verbraucht laut Zähler dauerhaft 550W. Im Portal wird mir 540W oder 600W angezeigt (s.Bild)
![cgfhgnghjjmgh](https://user-images.githubusercontent.com/128130987/225850736-7525ebb4-5192-4353-85ea-7e9ee652a2f0.JPG)

-Wie kann ich einstellen, das mir der Tagesverbrauch der WP nicht in Wh sondern in kW angezeigt wird?
![grergrfgeetg](https://user-images.githubusercontent.com/128130987/225850622-fde3a7f8-eb50-4ce2-8511-2efebb00b95a.JPG)
![jt6ugjhgjhjkhgm](https://user-images.githubusercontent.com/128130987/225850629-ba427c6e-9621-40a0-9a69-88fee6165b5a.JPG)

Danke

**Comment by camueller** [comment]
> Wieso springt der Verbrauch der WP im Sunnyportal?Die WP verbraucht laut Zähler dauerhaft 550W. Im Portal wird mir 540W oder 600W angezeigt

Du hast ja einen Modbus-Zähler - die liefern normalerweise sehr genauer Werte. Du kannst in der Log-Datei des SAE nachsehen, welche Werte der Zähler geliefert hat.

> Wie kann ich einstellen, das mir der Tagesverbrauch der WP nicht in Wh sondern in kW angezeigt wird?

Hat nichts mit dem SAE zu tun aber lässt sich im Sunny Portal wohl nicht einstellen.

**Comment by DE89UL** [comment]
Im Log finde ich den Punkt "power=540". Liegt es eventuell daran, das nach dem Parameter Zählerstand geguckt wird?


Ich habe es jetzt mal auf Leistung gestellt. Jetzt werden mir nur 151W angezeigt. Register ist wohl falsch. Ich habe den sdm72dm-v2. Muss ich das Register anpassen? Wenn ja wie genau funktioniert das?
![5656656](https://user-images.githubusercontent.com/128130987/225869699-9ef27b3a-79c2-441a-a2a4-991d84226428.JPG)
![343232323](https://user-images.githubusercontent.com/128130987/225869714-53f0d40c-5502-4c11-99c7-8f13751b83a1.JPG)

**Comment by camueller** [comment]
Du solltest `Zählerstand` verwenden.

Steht alles in der Doku: https://github.com/camueller/SmartApplianceEnabler/blob/master/doc/ModbusMeter_DE.md

**Comment by DE89UL** [comment]
Also waren meine Einstellung korrekt. Ich versteh nur nicht warum der Wert immer springt. Aktuell ist die WP aus und es werden 0W angezeigt. Wie wird mir der Standby im Portal angezeigt?

**Comment by camueller** [comment]
Was heisst "angezeigt"? Relevant ist, was der SAE aus dem Zähler ausliesst. Das wird an den SHM und indirekt an das Sunny Portal gemeldet.

**Comment by DE89UL** [comment]
Der Modbuszähler zeigt 10W Verbrauch der WP. Sunny Portal sagt 0W. Semp sagt auch 0. Liegt das am Register?
![3434343443565](https://user-images.githubusercontent.com/128130987/225888681-62ed9ccf-2fbd-4124-a5f5-f93179e5d479.JPG)

**Comment by camueller** [comment]
Schau doch einfach in die Logs und nicht auf irgendwelche Anzeigen.

**Comment by DE89UL** [comment]
Im Log steht auch "power 0"
